paddle/framework/channel_test.cc

```diff
@@ -200,10 +200,34 @@ TEST(Channel, UnbufferedLessReceiveMoreSendTest) {
     ch->Receive(&recv);
     EXPECT_EQ(recv, i);
   }
-  std::this_thread::sleep_for(std::chrono::milliseconds(100));  // wait 0.5 sec
+  std::this_thread::sleep_for(std::chrono::milliseconds(500));  // wait 0.5 sec
   EXPECT_EQ(sum_send, 3U);
 
   CloseChannel(ch);
   t.join();
   delete ch;
 }
+
+TEST(Channel, UnbufferedEmptyReceiveSendReceiveTest) {
```

Comment:
It seems that `UnbufferedEmptyReceiveSendReceiveTest` is similar with [`UnbufferedMoreReceiveLessSendTest`](https://github.com/PaddlePaddle/Paddle/pull/8061/files#diff-eb50f21fc86651780ef66f64c45711fbR211). Is there a redundancy? I'm not sure. @abhinavarora